### DISCUSSION TITLE
mps-platform-docs/platform_essentials/how_everything_is_connected/

### DISCUSSION THREAD

**Discussion by giscus** [mbeddr/mps-platform-docs#10] [General]
# mps-platform-docs/platform_essentials/how_everything_is_connected/

This website collects practical knowledge about MPS and popular MPS platforms such as MPS-extensions and mbeddr. It is written mainly in the form of a Q&A.

http://mbeddr.com/mps-platform-docs/platform_essentials/how_everything_is_connected/

**Comment by nolisj** [comment]
This is a very good overview of the relationships between MPS and Mbeddr.
